client/src/Components/SideBar.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import '../Media/CSS/App.css';
+
+export default function sideBar() {
+  return (
+    <div className='side-bar'></div>
+  )
```

Comment:
component only adding a div with class , manage it by directly adding class where required